tests/test_object_states.py

```diff
@@ -340,7 +339,7 @@ def test_temperature():
     assert dishtowel.states[Temperature].get_value() == m.object_states.temperature.DEFAULT_TEMPERATURE
 
     # Open the microwave
-    microwave.joints["j_link_0"].set_pos(np.pi / 2)
+    microwave.joints["joint_0"].set_pos(np.pi / 2)
```

Comment:
interesting. was this using the newest assets? i was testing on my local branch on friday and don't recall running into an error...(But maybe i was on the old assets?)

Reply:
this is using the old asset. Once we merge our two PRs. I will test Cem's new asset and make tests work.